src/krux/pages/__init__.py

```diff
@@ -146,19 +148,24 @@ def capture_from_keypad(
         """
         buffer = starting_buffer
         pad = Keypad(self.ctx, keysets, possible_keys_fn)
+        big_tittle = len(self.ctx.display.to_lines(title)) > 1
```

Comment:
typo "big_tittle" becomes "big_title" but not currently a bug, would need to get altered below too.